integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/SetDeviceCommunicationSettings.feature

```diff
@@ -13,7 +13,7 @@ Feature: SmartMetering Management - Set Device Communication Settings
       | DeviceIdentification     | TEST1024000000001 |
       | DeviceType               | SMART_METER_E     |
       | ChallengeLength          | 8                 |
-      | WithListSupported        | true              |
+      | WithListMax              | 32                |
```

Comment:
Since 32 was the default value used when with list supported was true I would feel comfortable with a second test with a different max value.

Reply:
Okay, this is covered by unit tests

Reply:
This test is for making sure that setting the value works. Testing here with another value is not necessary.
In the test for the DlmsHelper there are some tests with different values to verify the correct request(s) are sent.